xmsgrid/python/ugrid/XmUGridUtils_py.cpp

```diff
@@ -21,26 +21,76 @@ namespace py = pybind11;
 PYBIND11_DECLARE_HOLDER_TYPE(T, boost::shared_ptr<T>);
 
 void initXmUGridUtils(py::module &m) {
-    py::module modXmUGridUtils = m.def_submodule("XmUGridUtils")
-      .def("xm_read_ugrid_from_ascii_file", &xms::XmReadUGridFromAsciiFile,
-        "Read XmUGrid from an ASCII file.",py::arg("file_path"))
-      .def("xm_write_ugrid_to_ascii_file", &xms::XmWriteUGridToAsciiFile,
-        "Write an XmUGrid to an ASCII file.",py::arg("ugrid"),py::arg("file_path"))
-      .def("cross", [](py::iterable Origin, py::iterable A, py::iterable B) -> double {
+  py::module modXmUGridUtils = m.def_submodule("XmUGridUtils");
+  // ---------------------------------------------------------------------------
+  // function: xm_read_ugrid_from_ascii_file
+  // --------------------------------------------------------------------------- 
+  const char* xm_read_ugrid_from_ascii_file_doc = R"pydoc(
+    Read XmUGrid from an ASCII file.
+
+    Args: 
+        file_path (str): Path of the ascii file
+
+    Returns:
+        XmUGrid: The generated XmUGrid
+  )pydoc";
+  modXmUGridUtils.def("xm_read_ugrid_from_ascii_file", &xms::XmReadUGridFromAsciiFile,
+        xm_read_ugrid_from_ascii_file_doc,py::arg("file_path"));
+  // ---------------------------------------------------------------------------
+  // function: xm_write_ugrid_to_ascii_file
+  // --------------------------------------------------------------------------- 
+  const char* xm_write_ugrid_to_ascii_file_doc = R"pydoc(
+    Read XmUGrid from an ASCII file.
+
+    Args: 
+        ugrid (XmUGrid): XmUGrid to write to file
+        file_path (str): Path of the ascii file to be created
+  )pydoc";
+  modXmUGridUtils.def("xm_write_ugrid_to_ascii_file", &xms::XmWriteUGridToAsciiFile,
+        xm_write_ugrid_to_ascii_file_doc,py::arg("ugrid"),py::arg("file_path"));
```

Comment:
Do we want to get rid of the xm off of this function? None of the other functions use that prefix